apps/storybook/src/Dialog.stories.tsx

```diff
@@ -120,7 +120,98 @@ export const Modal: Story = () => {
             aliquip ex ea commodo consequat. Duis aute irure dolor in
             reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
             pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-            culpa qui officia deserunt mollit anim id est laborum.
+            culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum
```

Comment:
you'll need to update visual test images if youre changing the story

Reply:
I have reverted the change altogether